conda_build.recipe/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+
+# Set the version info based on python version.
+VERSION="$(python setup.py --version)"
+
+# Add the git hash if it is missing.
+if [[ "${VERSION}" == *+ ]];
+then
+    VERSION="${VERSION}$(git rev-parse --short HEAD)"
```

Comment:
Looking at ``git_version`` in ``setup.py``, I could imagine something is off about the language envvars that causes this to fail under some circumstances.